### PR TITLE
Making instructions for installation more clear

### PR DESCRIPTION
**Pull request by HegarGarcia**
Separating commands for each shell instead of notes to make instructions clearer.

Make sure you have checked all steps below.

### Prerequisite
* [x] Please consider implementing the feature as a hook script or plugin as a first step.
  * pyenv has some powerful support for plugins and hook scripts. Please refer to [Authoring plugins](https://github.com/pyenv/pyenv/wiki/Authoring-plugins) for details and try to implement it as a plugin if possible.
* [x] Please consider contributing the patch upstream to [rbenv](https://github.com/rbenv/rbenv), since we have borrowed most of the code from that project.
  * We occasionally import the changes from rbenv. In general, you can expect changes made in rbenv will be imported to pyenv too, eventually.
  * Generally speaking, we prefer not to make changes in the core in order to keep compatibility with rbenv.
* [x] My PR addresses the following pyenv issue (if any)
  - https://github.com/pyenv/pyenv/issues/XXXX

### Description
- [x] Here are some details about my PR
       Moving instruction notes to actual copy-paste shell commands.

### Tests
- [x] My PR adds the following unit tests (if any) - No tests needed

### CODE REVIEW

**Comment by GabrielBB** [comment]
Yup, I was going to open this PR

File: README.md [hunk 1]
```diff
@@ -228,13 +228,26 @@ easy to fork and contribute any changes back upstream.
...
+     ~~~
+   
+   - For **Zsh**:
+     ~~~ bash
+     $ echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.zshrc
+     ~~~
+     
+   - For **Fish**:
+     ~~~ bash
+     $ echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  pyenv init - | source"\nfi' >> ~/.zshrc
```
Comment (enyrgy) [inline README.md:248]: Wouldn't you want to echo this to `~/.config/fish/config.fish` instead of `~/.zshrc` or am I mistaken?

Comment (HegarGarcia) [reply README.md]: You were right the commands wasn't correct! Fixed it with the latest one shown in this thread: (https://github.com/pyenv/pyenv/issues/32). Tried myself and runs perfectly!

**Comment by anton-petrov** [comment]
Already fixed.

**Pull request closed without merging**